ansible/roles/openshift_outbound_tcp_logging/tasks/main.yml

```diff
@@ -43,42 +43,48 @@
       after: "id_outbound_tcp_log_4_"
       regex: "id_outbound_tcp_log_5_"
 
-    forward_logging_rules:
+    filtering_rules:
     # Matches first packet in new outbound TCP connections, -A for use in /etc/sysconfig/iptables.
-    - chain: FORWARD
-      line:  "-A FORWARD -o eth0 -p tcp ! -d 172.16.0.0/12 -m state --state NEW -m comment --comment id_outbound_tcp_log_7_ -j oo-outbound-logging"
+    - chain: OPENSHIFT-OUTBOUND-FILTERING
+      line:  "-A OPENSHIFT-OUTBOUND-FILTERING -o eth0 -p tcp ! -d 172.16.0.0/12 -m state --state NEW -m comment --comment id_outbound_tcp_log_7_ -j oo-outbound-logging"
       after: "^-A INPUT"
       regex: "id_outbound_tcp_log_7_"
 
-    forward_logging_memory:
+    filtering_memory:
     # Matches first packet in new outbound TCP connections, -I to send to the top of the FORWARD chain in memory.
-    - chain: FORWARD
-      line:  "-I FORWARD -o eth0 -p tcp ! -d 172.16.0.0/12 -m state --state NEW -m comment --comment id_outbound_tcp_log_7_ -j oo-outbound-logging"
+    - chain: OPENSHIFT-OUTBOUND-FILTERING
+      line:  "-I OPENSHIFT-OUTBOUND-FILTERING -o eth0 -p tcp ! -d 172.16.0.0/12 -m state --state NEW -m comment --comment id_outbound_tcp_log_7_ -j oo-outbound-logging"
       after: "^-A INPUT"
       regex: "id_outbound_tcp_log_7_" 
 
-- name: list iptables rules FORWARD chain
-  command: '/usr/sbin/iptables -w -nL FORWARD'
+- name: list iptables rules filtering chain
+  command: '/usr/sbin/iptables -w -nL OPENSHIFT-OUTBOUND-FILTERING'
   changed_when: False
-  register: forwardchain
+  register: filteringchain
+  ignore_errors: True
 
 - name: list iptables rules logging chain
   command: '/usr/sbin/iptables -w -nL oo-outbound-logging'
   changed_when: False
   register: loggingchain
   ignore_errors: True
 
-- name: add in-memory iptables chain
+- name: add in-memory filtering chain
+  command: "/usr/sbin/iptables -N OPENSHIFT-OUTBOUND-FILTERING"
+  when: not "OPENSHIFT-OUTBOUND-FILTERING" in filteringchain.stdout
+  with_items: "{{ outbound_logging_chain }}"
```

Comment:
This `with_items` seems vestigial. You never refer to {{ item }} in the rule, so I'm guessing you don't want the `with_items`.